e2e-tests/gef/cypress.config.js

```diff
@@ -27,8 +27,8 @@ module.exports = defineConfig({
   pageLoadTimeout: 120000,
   responseTimeout: 120000,
   numTestsKeptInMemory: 1,
-  viewportWidth: 3840,
-  viewportHeight: 2400,
+  viewportWidth: 2000,
```

Comment:
I would recommend 16:9 so change that to 1920 x 1080 please.

Reply:
done